src/State/PlaylistProvider.php

```diff
@@ -38,13 +38,15 @@ public function toOutput(object $object): PlaylistDTO
                 'duration' => $schedule->getDuration(),
             ];
         }
-        $output->schedules = $schedulesOutput;
+        $output->schedules = 0 === count($schedulesOutput) ? null : $schedulesOutput;
```

Comment:
Why change from array to null?

Reply:
If we set the value to `null` for empty arrays the field does not get serialized in the API output. This makes it consistent with the behavior for non-array fields with the new default:
```
normalization_context:
    # Default value to omit null values in conformance with the JSON Merge Patch RFC.
    skip_null_values: true
```

Reply:
Ok.
This will require a change in the client here: https://github.com/os2display/display-client/blob/develop/src/service/scheduleService.js#L191

Reply:
Reverted